primap2/_aggregate.py

```diff
@@ -309,23 +309,23 @@ def add_aggregates_coordinates(
                         rule_tolerance = rule.pop("tolerance")
                     else:
                         rule_tolerance = tolerance
-                    if "filter" in rule.keys():
-                        filter_ = rule.pop("filter")
-                        if "variable" in filter_.keys():
-                            if da_out.name in filter_["variable"]:
-                                filter_.pop("variable")
+                    if "sel" in rule.keys():
+                        sel_ = rule.pop("sel")
```

Comment:
```suggestion
                        sel = rule.pop("sel")
```
`filter` is a python reserved keyword (well, just a built-in function name, to be precise), that's the only reason we wrote `filter_` with the `_` at the end. Since `sel` is not a built-in function name, we can simply call it `sel`.

Reply:
Will do, was lazy. the variable was called `filter_` and I just replaced `filter` with `sel`